src/ECS/entity/EntityManager.hpp

```diff
@@ -35,6 +35,21 @@ namespace ECS {
                 return __entities.back();
             }
 
+            /**
+             * @brief Spawn a new entity
+             */
+            Entity spawnEntityWithId(size_t entity) {
+                // if (std::find(__entities.begin(), __entities.end(),e)!=__entities.end()) {
+                //     std::cerr << "Error: Entity with id n°" << e << " already exists" << std::endl;
+                //     return;
+                // }
+                while (entity >= __entities.size()) {
+                    __entities.push_back(Entity(__nextEntityId++));
+                }
+                __entities.insert(__entities.begin() + entity, Entity(entity));
+                return __entities.at(entity);
+            }
+
```

Comment:
@Tugduoff do you use this function ? If not we can delete it.

Reply:
I don't, we can safely delete it
![image](https://github.com/user-attachments/assets/131ca4eb-eba3-4fbb-b8e1-c8cad67dda6e)

Reply:
I removed it then :saluting_face: